hpOneView/resources/storage/volumes.py

```diff
@@ -52,6 +52,103 @@ def __init__(self, con):
             "type": "Snapshot"
         }
 
+    def get_all(self, start=0, count=-1, filter='', sort=''):
+        """
+        Gets a paginated collection of managed volumes. The collection is based on optional
+        sorting and filtering, and constrained by start and count parameters.
+
+        Args:
+            start:
+                The first item to return, using 0-based indexing.
+                If not specified, the default is 0 - start with the first available item.
+            count:
+                The number of resources to return. A count of -1 requests all the items.
+                The actual number of items in the response may differ from the requested
+                count if the sum of start and count exceed the total number of items, or
+                if returning the requested number of items would take too long.
+            filter:
+                A general filter/query string to narrow the list of items returned. The
+                default is no filter - all resources are returned.
+            sort:
+                The sort order of the returned data set. By default, the sort order is based
+                on create time, with the oldest entry first.
+
+        Returns: A list of volumes.
+
+        """
+        return self._client.get_all(start, count, filter=filter, sort=sort)
+
+    def get(self, id_or_uri):
+        """
+        Gets the managed volume.
+
+        Args:
+            id_or_uri: Could be either the volume id or the volume uri.
+
+        Returns: Managed volume.
+        """
+        return self._client.get(id_or_uri)
+
+    def get_by(self, field, value):
+        """
+        Get all managed volumes that matches the given filter.
+        The search is case insensitive.
+
+        Args:
+            field: Field name to filter.
+            value: Value to filter.
+
+        Returns: A list of volumes.
+
+        """
+        return self._client.get_by(field, value)
+
+    def create(self, resource, timeout=-1):
+        """
+        Creates a Volume.
+
+          It's possible to create the volume in 6 different ways:
+          1) Common = Storage System + Storage Pool
+          2) Template = Storage Volume Template
+          3) Common with snapshots = Storage System + Storage Pool + Snapshot Pool
+          4) Management = Storage System + wwn
```

Comment:
I think items 4) and 5) need a special note that they are for adding volumes for management and that they do not create new volumes.